numba/tests/test_stencils.py

```diff
@@ -427,6 +427,37 @@ def test_impl_seq(n):
         n = 100
         self.check(test_impl_seq, test_impl, n)
 
+    @skip_unsupported
+    @tag('important')
+    def test_stencil_call_const(self):
```

Comment:
Add a test with a[-c + 1] and test for our intended behavior.

Reply:
done.

Reply:
actually, just `a[1-c]` is supported since this is not recursive.

Reply:
hmmmm, I think this may be confusing for users, if the `a[-c + 1]` case is used, then the error message is:
```
ValueError: Failed in nopython mode pipeline (step: convert to parfors)
Variable stencil index only possible with known neighborhood
```
which I think it just the indexing problem catch standard response? This adds a bit to the confusion given `a[1 - c]` works fine. 

What do you think about this? I understand why the `1-c` case is easier, any idea of the effort to support the former? Thanks.

Reply:
Yes, it's the standard response. I just added support for the recursive cases as well.